build/builder.go

```diff
@@ -573,6 +582,18 @@ func (b *Builder) Finish() error {
 	b.finishedShards = map[string]string{}
 
 	for p := range toDelete {
+		if _, err := os.Stat(filepath.Join(filepath.Dir(p), zoekt.TombstoneFileName)); err == nil {
```

Comment:
minor: when I first read this I thought you where checking if the tombstone existed for the shard. Rather than exporting `zoekt.TombstoneFileName` can you maybe add a method like `zoekt.TombstonesEnabled(dir string) bool`? This would read clearer and avoid copy pasta on os.Stat.